renderer/components/UI/CurrencyFieldGroup.js

```diff
@@ -88,7 +99,7 @@ const CurrencyFieldGroup = React.forwardRef(
                 isRequired={isRequired}
                 label={cryptoLabel || <FormattedMessage {...messages.amount} />}
                 name="amountCrypto"
-                onChange={handleAmountCryptoChange}
+                onValueChange={handleAmountCryptoChange}
```

Comment:
It's no longer possible to enter a number into the crypto amount field on the pay form. I have a feeling it may be related to this change. I think it causes some kind conflict with other onchange handlers that kick in as part of the number field handling.